DockerHubfile

```diff
@@ -0,0 +1,29 @@
+#### Create Jar 
+FROM gradle:jdk8 as builder 
+ 
+COPY --chown=gradle:gradle . /home/gradle/app 
```

Comment:
can't do this docker hub automated builds don't support it :(

Reply:
this will work instead https://github.com/hmcts/cmc-claim-store/pull/320/files#diff-1a262826143d213da9e58fe82c06a2acR3